core/src/main/java/org/polypheny/db/algebra/enumerable/EnumerableScan.java

```diff
@@ -255,7 +271,7 @@ public Result implement( EnumerableAlgImplementor implementor, Prefer pref ) {
 
 
     @Override
-    public AlgOptCost computeSelfCost( AlgOptPlanner planner, AlgMetadataQuery mq ) {
+    public AlgOptCost computeSelfCost( AlgPlanner planner, AlgMetadataQuery mq ) {
         if ( Catalog.mode == RunMode.TEST ) {
```

Comment:
Is it a good idea to handle costs differently depending on whether we run tests?

Reply:
This is done to allow various test to construct valid algebras. While we could remove this explicit adjustment here, and adjust it via reflections. This is in my opinion much more transparent than other approaches.

Reply:
Ok, maybe add this as a javadoc.